### ISSUE TITLE
Issue when extracting an archive to a folder that already exists

### ISSUE THREAD

**Issue by FerryJansen** [xebialabs-community/xld-folder-plugin#10]
When execution a Create action and the contents of the archive already exists, the process does nothing. I suspect it is awaiting a confimatrion to overwrite or not. This is the case when i manually execute it.

I propose to update the code from and to this:

Currentcode
`$shell.Namespace($destination).copyhere($item)`

New code
`$shell.Namespace($destination).copyhere($item,16)`
This will anwser "Yes" to any prompt.

Do you think this is a good fix?

**Comment by jdewinne** [comment]
Makes sense (according to the [docs](https://docs.microsoft.com/en-us/windows/desktop/api/shldisp/nf-shldisp-folder-copyhere). You wanna create a PR?

**Comment by FerryJansen** [comment]
I think i just did. First time so wasn't too sure I was doing it right.
